src/reference/supported-platforms.md

```diff
@@ -27,7 +27,7 @@ make sure to make a corresponding change in the get started pages:
   * /src/get-started/install/macos.md
   * /src/get-started/install/windows.md
   * /src/get-started/install/linux.md
-{% endcomment %}
+{% endcomment -%}
```

Comment:
I'm curious what this change does

Reply:
It just removes some extra whitespace in the generated HTML around where the comment would be.

I primarily made the change to be consistent with comments elsewhere.